### DISCUSSION TITLE
Off-topic question

### DISCUSSION THREAD

**Discussion by Troy-peng-0327** [fcakyon/small-object-detection-benchmark#14] [Q&A] [answered]
Good afternoon. First of all, congratulations on the success of your excellent project. I had an off-topic question because I noticed your question in mmdetection. Have you tried to implement Querydet at mmdetection? I had troubles when I implemented it. Would you mind sharing and discussing them together if I could?

**Comment by fcakyon** [comment, marked as answer]
Thanks for your support!

Unfortunately, I did not try to implement querydet and not sure if I can help you.
